src/staticdata_utils.c

```diff
@@ -1022,18 +1022,18 @@ static int jl_verify_graph_edge(size_t *maxvalids2_data, jl_array_t *edges, size
         if (idx != childidx) {
             if (max_valid < maxvalids2_data[childidx])
                 maxvalids2_data[childidx] = max_valid;
-            if (_jl_debug_method_invalidation && max_valid != ~(size_t)0) {
-                jl_method_instance_t *mi = (jl_method_instance_t*)jl_array_ptr_ref(edges, childidx * 2);
-                jl_value_t *loctag = NULL;
-                JL_GC_PUSH1(&loctag);
-                jl_array_ptr_1d_push(_jl_debug_method_invalidation, (jl_value_t*)mi);
-                loctag = jl_cstr_to_string("verify_methods");
-                jl_array_ptr_1d_push(_jl_debug_method_invalidation, loctag);
-                jl_array_ptr_1d_push(_jl_debug_method_invalidation, (jl_value_t*)cause);
-                JL_GC_POP();
-            }
         }
         visited->items[childidx] = (void*)1;
+	if (_jl_debug_method_invalidation && max_valid != ~(size_t)0) {
```

Comment:
The indention looks off (tabs versus spaces?)

Reply:
Thanks! I was on a machine that I'd evidently not fully set up for Julia/C development.